11LessonEleven/in-class-project/index.html

```diff
@@ -15,28 +15,28 @@
     <header class="header">
         <h1>Welcome to our site</h1>
     </header>
-    <main class="clearfix">
+    <main class="clearfix main">
 
         <div class="box">
             <div class="blue"></div>
             <div class="container">
                 <h1>Title</h1>
-                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. Biscuit danish jelly-o jelly-o cupcake candy powder chocolate pie. </p>
+                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. </p>
             </div>
         </div>
 
         <div class="box">
             <div class="orange"></div>
             <div class="container">
                 <h1>Title</h1>
-                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. Biscuit danish jelly-o jelly-o cupcake candy powder chocolate pie. </p>
+                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. </p>
           </div>
         </div>
         <div class="box">
             <div class="aqua"></div>
             <div class="container">
                 <h1>Title</h1>
-                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. Biscuit danish jelly-o jelly-o cupcake candy powder chocolate pie. </p>
+                <p>Pudding sweet roll jelly-o chupa chups cheesecake lollipop muffin cake. Cupcake muffin chocolate bar soufflé donut sesame snaps toffee jujubes. </p>
             </div>
         </div>
     </main>
```

Comment:
Hi Luke,

Just checking my grades -

Lesson 7 in class 0?
Lesson 9 in class 50?

I'm missing something ... I did both projects and turned them in

Still waiting to hear back from github ...

Barbara
--------------------------------------------
On Tue, 9/13/16, Ethan Stenis <notifications@github.com> wrote:

 Subject: Re: [AustinCodingAcademy/intro-workbook] Austin.intro.2.Dukette.Barbara (#92)
 To: "AustinCodingAcademy/intro-workbook" <intro-workbook@noreply.github.com>
 Cc: "Bjdukette" <Bjdukette@yahoo.com>, "Author" <author@noreply.github.com>
 Date: Tuesday, September 13, 2016, 6:08 PM
 
 In 11LessonEleven/in-class-project/index.html:
 >            </div>
 >          </div>
 >          <div class="box">
 >              <div
 class="aqua"></div>
 >              <div class="container">
 >                  <h1>Title</h1>
 > -                <p>Pudding sweet roll jelly-o
 chupa chups cheesecake lollipop muffin cake. Cupcake muffin
 chocolate bar soufflé donut sesame snaps toffee jujubes.
 Biscuit danish jelly-o jelly-o cupcake candy powder
 chocolate pie. </p>
 > +                <p>Pudding sweet roll jelly-o
 chupa chups cheesecake lollipop muffin cake. Cupcake muffin
 chocolate bar soufflé donut sesame snaps toffee jujubes.
 </p>
 >              </div>
 >          </div>
 >      </main>
 
 Just need some CSS on that H1 in the phone media query.
 Let's get that fixed and I can bump your grade up.
 
 
 
 —
 You are receiving this because you authored the thread.
 Reply to this email directly, view
 it on GitHub, or mute
 the thread.